src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetCohortPriorLearningError/GetCohortPriorLearningErrorQueryHandlerTests.cs

```diff
@@ -0,0 +1,298 @@
+                .SetMaxFundingBandForStandard(1, 5000)
+                .SetApprenticeshipPriorLearningData(trainingTotalHours, priorLearning);
+
+            var result = await fixture.Handle();
+
+            Assert.AreEqual(expectedToHaveAValue, result.PercentageOfPriorLearning.HasValue);
+            Assert.AreEqual(expectedToHaveAValue, result.MinimumPercentageReduction.HasValue);
+            Assert.AreEqual(expectedToHaveAValue, result.MinimumPriceReduction.HasValue);
+            Assert.IsFalse(result.RplPriceReductionError);
+        }
+
+        [TestCase(3200, 2000, false)]
+        [TestCase(8200, 800, true)]
+        public async Task Handle_WhenApprenticeshipFounaxFundingFoundForFramework_ThenRPLSummaryDataShouldBeReturnedWithCorrectValues(
+            int maxFundingBand, int trainingTotalHours, bool reductionIsInError)
+        {
+            var priorLearning = new ApprenticeshipPriorLearning
+            {
+                DurationReducedByHours = 200,
+                PriceReducedBy = 1000,
+                IsDurationReducedByRpl = true,
+                DurationReducedBy = null
+            };
+
+            var fixture = new GetCohortPriorLearningErrorQueryHandlerTestsFixtures()
+                .SetApprentice(ProgrammeType.Framework, "123-123", DateTime.Today)
+                .SetMaxFundingBandForFramework("123-123", maxFundingBand)
+                .SetApprenticeshipPriorLearningData(trainingTotalHours, priorLearning);
+
+            var result = await fixture.Handle();
+
+            Assert.AreEqual(maxFundingBand, result.FundingBandMaximum);
+            Assert.AreEqual((decimal)priorLearning.DurationReducedByHours / trainingTotalHours * 100, result.PercentageOfPriorLearning);
+            Assert.AreEqual((decimal)priorLearning.DurationReducedByHours / trainingTotalHours * 100 / 2, result.MinimumPercentageReduction);
+            Assert.AreEqual(reductionIsInError, result.RplPriceReductionError);
+        }
+    }
+
+    public class GetCohortPriorLearningErrorQueryHandlerTestsFixtures
+    {
+        public ProviderCommitmentsDbContext Db { get; set; }
+        public Mock<IFeatureTogglesService<FeatureToggle>> FeatureToggleServiceMock { get; set; }
+        public GetDraftApprenticeshipPriorLearningSummaryQueryHandler Handler { get; set; }
+        public ApprenticeshipPriorLearning PriorLearning { get; set; }
+        public FlexibleEmployment FlexibleEmployment { get; set; }
+
+        private long CohortId = 1;
+        private long ApprenticeshipId = 1;
+
+        public GetCohortPriorLearningErrorQueryHandlerTestsFixtures()
+        {
+            Db = new ProviderCommitmentsDbContext(new DbContextOptionsBuilder<ProviderCommitmentsDbContext>().UseInMemoryDatabase(Guid.NewGuid().ToString()).Options);
+
+            Handler = new GetDraftApprenticeshipPriorLearningSummaryQueryHandler(
+                new Lazy<ProviderCommitmentsDbContext>(() => Db), new RplFundingCalulationService());
```

Comment:
bcause we are testing RplFundingCalulationService in the other TESTs via the original draftApprenticeshipPriorLearningSummaryHandler (I found it) you don't need to test the implementation of RplFundingCalulationService you could just mock that service here